lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -1451,6 +1464,8 @@ class MailboxDashBoardController extends ReloadableController {
         () => _replaceBrowserHistory(uri: result.value2)
       );
     }
+
+    _getAllIdentities();
```

Comment:
why we need `getAllIdentity` when go to settings?

Reply:
Not so. We only get it back when we return to the `dashboard` screen from settings. Please look above, you will see we have to `await push` to wait for a callback from the settings screen.